templates/zerver/api/common-error-payloads.md

```diff
@@ -19,3 +19,12 @@ A typical failed JSON response for when the user is not authorized
 for a query:
 
 {generate_code_example|user-not-authorized-error|fixture}
+
+## Invalid stream name
+
+A typical failed JSON response for when an invalid stream name is
+passed to a query that expects a stream name as an argument (such as
+[`client.add_subscriptions`](/api/add-subscriptions),
+[`client.remove_subscriptions`](/api/remove-subscriptions), etc.):
+
+{generate_code_example|invalid-stream-error|fixture}
```

Comment:
I'm not sure it makes sense to me to highlight this as a top-level thing, given that only a couple endpoints use it.

Reply:
Are you saying we should move it out of common-error-payloads and just put them in the specific files that they relate to?